### PR TITLE
IDEMPIERE-6335 - Create parameter to specify document type in material receipt

### PR DESCRIPTION
**Pull request by VitorVilla**
https://idempiere.atlassian.net/browse/IDEMPIERE-6335

# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] I have checked my code and corrected any misspellings
- [ ] In hard-to-understand areas, I have commented my code.
- [ ] My changes generate no new warnings
### Tests
- [ ] I have tested the direct scenario that my code is solving
- [ ] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
### Documentation
- [ ] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

**Comment by VitorVilla** [comment]
"Could you bring this to version 11?"

File: migration/iD11/oracle/202412041007_IDEMPIERE-6335.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-- IDEMPIERE-6335
+SELECT register_migration_script('202412041007_IDEMPIERE-6335.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Dec 4, 2024, 10:07:59 AM BRT
+INSERT INTO AD_Process_Para (AD_Process_Para_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,Name,Description,Help,AD_Process_ID,SeqNo,AD_Reference_ID,IsRange,AD_Val_Rule_ID,FieldLength,IsMandatory,ColumnName,IsCentrallyMaintained,EntityType,AD_Element_ID,AD_Process_Para_UU,IsEncrypted,IsAutocomplete,DateRangeOption,IsShowNegateButton) VALUES (nextidfunc(200,'N'),0,0,'Y',TO_TIMESTAMP('2024-12-04 10:07:59','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2024-12-04 10:07:59','YYYY-MM-DD HH24:MI:SS'),100,'Document Type','Document type or rules','The Document Type determines document sequence and processing rules',142,20,19,'N',52054,0,'N','C_DocType_ID','Y','U',196,'2c4cf92c-8326-4fa5-8e0c-8643bf05a2e1','N','N','D','N')
```
[the remark this answers is not in the document]
Comment (VitorVilla) [reply migration/iD11/oracle/202412041007_IDEMPIERE-6335.sql]: I adjusted it to use the centralizer

**Review by CarlosRuiz-globalqss — CHANGES_REQUESTED** [review]
can you please apply the following patch
[pr2581PR0.patch.txt](https://github.com/user-attachments/files/18099949/pr2581PR0.patch.txt)

I tested this successfully, so, is just needing that to approve it.

**Comment by VitorVilla** [comment]
patch applied

**Review by CarlosRuiz-globalqss — APPROVED** [review]
tested

**Pull request merged**